lib/tracing-limit/src/lib.rs

```diff
@@ -474,11 +495,11 @@ mod test {
         let events: Arc<Mutex<Vec<String>>> = Default::default();
 
         let recorder = RecordingLayer::new(Arc::clone(&events));
-        let sub =
-            tracing_subscriber::registry::Registry::default().with(RateLimitedLayer::new(recorder));
+        let sub = tracing_subscriber::registry::Registry::default()
```

Comment:
👍 to updating this test. Could we add another test that asserts the ability to override the default limit at the call site? You could just copy this test, change the default rate limit to be high, but then set ` internal_log_rate_secs = 1` at the callsite.